docs/UserGuide.md

```diff
@@ -70,52 +68,56 @@ While exploring PoochPlanner’s features with this user guide, you can take not
 
 Symbol | Meaning
 --------|------------------
-:information_source: | Important Information
-:exclamation: | Warning or Caution
-:bulb: | Additional Information such as Tips or Notes
+:information_source: | Important information
+:exclamation: | Warning or caution
+:bulb: | Additional information such as tips or notes
 
-The following glossary table provides clarification on commonly-used terms as well as terminology that is possibly unknown to you.
+The following glossary table provides clarification on commonly-used terminology.
 
 Phrases | Meaning
 --------|------------------
-GUI | GUI stands for Graphical User Interface and it represents the visual display of PoochPlanner that you can see when the application is run.
-GUI Component | A subsection of the GUI. For more information on specific GUI components, refer to this [section](#navigating-the-gui).
+GUI | GUI stands for Graphical User Interface and it represents the visual display of PoochPlanner that you see when the application is run.
+GUI component | The parts that make up a GUI. For more information on specific GUI components, refer to this [section](#navigating-the-gui).
 CLI | CLI stands for Command Line Interface and it represents a text-based user interface to interact with the application.
 Command | An input from the user that tells PoochPlanner to perform an action. View PoochPlanner’s [Command Summary](#command-summary).
-Prefix | Prefixes are like fields in a form you are required to fill up. They are information needed to be passed together with the command so that it can be executed. View PoochPlanner’s [Prefix Summary](#prefix-summary).
-Case-Sensitive | The casing of the alphabetic characters matters (e.g. “good” is different from “GOOD”).
-Case-Insensitive | The casing of the alphabetic characters does not matter (e.g. “good” is taken to be equal to “GOOD”).
+Prefix | Prefixes are like fields in a form you are required to fill up. They capture important information that is needed to execute the command. View PoochPlanner’s [Prefix Summary](#prefix-summary).
+Case-sensitive | The casing of the alphabetic characters matters (e.g. “good” is different from “GOOD”).
+Case-insensitive | The casing of the alphabetic characters does not matter (e.g. “good” is taken to be equal to “GOOD”).
+Space-sensitive | The number of spaces in a command matters (e.g. “happy puppy” is different from “happypuppy”).
+Space-insensitive | The number of spaces in a command does not matter (e.g. “happy puppy” is taken to be equal to “happypuppy”).
+Delimeter | A seperator in the command input that defines the boundaries between information in a command. ";" is a delimeter for our commands.
+Contact card | The box displayed for each contact, containing all their details.
 
 
 ## Quick start
 
-1. Ensure you have Java `11` installed in your Computer.
+1. Ensure you have Java `11` installed in your computer.
```

Comment:
"installed _on_" is more commonly used than "installed _in_"

Reply:
changed